recipe/meta.yaml

```diff
@@ -30,18 +30,18 @@ source:
     - patches/maxima.system.patch
 
 build:
-  number: 2
+  number: 3
   skip: true  # [win]
 
 requirements:
   build:
-    - toolchain
```

Comment:
I think you need a compiler as `ecl` is a source-to-source compiler which translates lisp to c.